respawn_altering/healer

```diff
@@ -1,6 +1,5 @@
 Healer- (5)
-- During the night, can reduce a person's respawn by 4 phases.
+- During the night, the Healer can tend to a respawning players wounds, speeding up their respawn progress by 4 phases.
```

Comment:
Shouldn't this also have can make the respawn time negative? Or is that not required?

Reply:
no because it only adds progress, and doesn't subtract. for example (0/4) progress to (2/4).

Reply:
Ohhh sorry, I misunderstood this whole thing. Should it appear for Finisher then?

Reply:
finisher is different, it changes the required amount needed. for example: changes 2 phases needed to respawn into 6 phases

Reply:
the main reason i didnt do it like that for wizard because it would just take too many lines to explain

Reply:
Alright that's fair, I think everything in this pr is done then.